src/reports.coffee

```diff
@@ -84,7 +87,35 @@ sendReports = (job, flag, done) ->
             report.type = 'Daily'
           else
             report.type = 'Weekly'
+
+          report.instance = config.alerts.himInstance
+
           try
+            i = 0
+            for data in report.data
+              do (data) ->
+                colorGrey = 'color: grey;'
+                rowColor = 'background-color: #d9ead3'
+                if i % 2
+                  rowColor = 'background-color: #b6d7a8;'
+
+                report.data[i].load = (if data.data[0]?.load? then data.data[0].load else 0)
+                report.data[i].avgResp = (if data.data[0]?.avgResp? then data.data[0].avgResp else 0)
+                report.data[i].failed = (if data.statusData[0]?.failed? then data.statusData[0].failed else 0)
+                report.data[i].successful = (if data.statusData[0]?.successful? then data.statusData[0].successful else 0)
+                report.data[i].processing = (if data.statusData[0]?.processing? then data.statusData[0].processing else 0)
+                report.data[i].completed = (if data.statusData[0]?.completed? then data.statusData[0].completed else 0)
+                report.data[i].completedWErrors = (if data.statusData[0]?.completedWErrors? then data.statusData[0].completedWErrors else 0)
+                report.data[i].loadStyle = (if report.data[i].load > 0 then '' else colorGrey)
+                report.data[i].avgRespStyle = (if report.data[i].avgResp > 0 then '' else colorGrey)
+                report.data[i].failedStyle = (if report.data[i].failed > 0 then 'color: red;' else colorGrey)
+                report.data[i].successfulStyle = (if report.data[i].successful > 0 then '' else colorGrey)
+                report.data[i].processingStyle = (if report.data[i].processing > 0 then '' else colorGrey)
+                report.data[i].completedStyle = (if report.data[i].completed > 0 then 'color: orange;' else colorGrey)
+                report.data[i].completedWErrorsStyle = (if report.data[i].completedWErrors > 0 then 'color: orangered;' else colorGrey)
+                report.data[i].rowColor = rowColor
```

Comment:
It would be better to keep all display stuff out of the data as its good practice, is it not possible to do this with pure css. If its hard then I'm ok with this way.